deploy-grafana.sh

```diff
@@ -18,13 +18,7 @@ docker run --detach \
     -p 0.0.0.0:3000:3000 \
     -v ~/sourcegraph-docker/grafana-disk:/var/lib/grafana \
     -v $(pwd)/grafana:/sg_config_grafana/provisioning/datasources \
-    -e GF_AUTH_ANONYMOUS_ENABLED=true \
-    -e GF_AUTH_ANONYMOUS_ORG_NAME='Main Org.' \
-    -e GF_AUTH_ANONYMOUS_ORG_ROLE=Editor \
-    -e GF_USERS_ALLOW_SIGN_UP='false' \
-    -e GF_USERS_AUTO_ASSIGN_ORG='true' \
-    -e GF_USERS_AUTO_ASSIGN_ORG_ROLE=Editor \
-    sourcegraph/grafana:6.3.3-1@sha256:bbfa5565f3755ac961c8560e94c323cc588fc241364c4c6839376ccdb7c756db
+    sourcegraph/grafana:6.3.3-2@sha256:f84e4260dce4e36f5d5e9c6deb393ac1e9c930ffae622c432fa86640a1f2699f
```

Comment:
Before merging this, please add back in the containers.json dashboard here.

Reply:
Oh, yes, yes. I'm an idiot. Of course. I'll add it back

Reply:
done